cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/ETLSpark.java

```diff
@@ -109,6 +109,9 @@ public void initialize() throws Exception {
     // to make sure fields that are the same but different casing are treated as different fields in auto-joins
     // see CDAP-17024
     sparkConf.set("spark.sql.caseSensitive", "true");
+    //For spark 3.2 onwards the spark.hadoopRDD.ignoreEmptySplits is set to true, which created ZERO stages while using
+    //database plugins : see PLUGIN-1613
+    sparkConf.set("spark.hadoopRDD.ignoreEmptySplits", "false");
```

Comment:
Do we need similar fix in replication to get the same drivers working there? @sumitjnn

Reply:
Replication doesn't use `spark.hadoopRDD` , so Replication should not be related to this change.